homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -14,6 +14,9 @@
 DEFAULT_NAME = "Xiaomi Miio Sensor"
 DATA_KEY = "sensor.xiaomi_miio"
 
+MODEL_AIRQUALITYMONITOR_V1 = "zhimi.airmonitor.v1"
```

Comment:
Please separate words with underscore in variable names.
```py
MODEL_AIR_QUALITY_MONITOR_V1 = "zhimi.airmonitor.v1"
```

Reply:
Done.